src/Stepic/Palindrome.java

```diff
@@ -1,18 +1,26 @@
 package Stepic;
-
+//this task is 2.3 from the stepik site.
+//Implement a method that checks if the given string is a palindrome.
+//Madam, I'm Adam!
 public class Palindrome {
     public static void main(String[] args) {
-         boolean palin;
+         boolean palin=true;
         String text = "Madam, I'm Adam!";
```

Comment:
what should do this variable 'text' ???

Reply:
since this code was written for verification and subsequent insertion into the stepic site compiler, this variable simulates the input data that comes to the class